### DISCUSSION TITLE
Branch setup failed error - PostGIS

### DISCUSSION THREAD

**Discussion by Pierre-Saigot** [supabase/supabase#22645] [Questions] [answered]
I have an issue when setting up a new branch on my Supabase project. This error is due to 'PostGIS' and 'pgaudit.' The problem is that pgaudit is not enabled in my extensions. Do you have any solutions? I'll attach a screenshot of my error.

![ScreenShot](https://github.com/supabase/supabase/assets/10871790/7b6a07fa-9dbf-4dcd-bc18-7d94b73df457)

`ERROR: PostGIS installation stopped: pgaudit.log is set to 'ddl'. Set pgaudit.log to 'none' before installing PostGIS. You may re-enable pgaudit after installation is complete. (SQLSTATE P0001)
At statement 17: CREATE EXTENSION IF NOT EXISTS "postgis" WITH SCHEMA "extensions"`

**Comment by sweatybridge** [comment, marked as answer]
Closing with a suggested workaround https://github.com/supabase/supabase/issues/22646#issuecomment-2049653471
